lib/rubygems/version.rb

```diff
@@ -279,7 +279,10 @@ def encode_with coder # :nodoc:
   # A version is considered a prerelease if it contains a letter.
 
   def prerelease?
-    @prerelease ||= !!(@version =~ /[a-zA-Z]/)
+    unless defined?(@prerelease)
```

Comment:
I'd prefer not to, since that is a method and it takes a string or symbol rather than the variable itself.